libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -1362,5 +1362,105 @@
+    },
+    "properties.comparison.calculationType.as.ratio.info.useIn._metric": {
+        "value": "To calculate what part of secondary metric value is primary metric value.",
+        "comment": "The content describes the pertinent scenarios for the ratio calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.ratio.info.useIn._measure": {
+        "value": "To calculate what part of secondary metric value is primary measure value.",
+        "comment": "The content describes the pertinent scenarios for the ratio calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.difference.info.useIn._metric": {
+        "value": "To calculate absolute difference between primary and secondary metrics values.",
+        "comment": "The content describes the pertinent scenarios for the difference calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.difference.info.useIn._measure": {
+        "value": "To calculate absolute difference between primary and secondary measures values.",
+        "comment": "The content describes the pertinent scenarios for the difference calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.info.formula.title": {
+        "value": "Formula",
+        "comment": "The title of the section that describes the formula for the calculation",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.change.info.formula": {
+        "value": "(Primary - Secondary) / Secondary",
+        "comment": "The content describes the formula of the change calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.ratio.info.formula": {
+        "value": "Primary / Secondary",
+        "comment": "The content describes the formula of the ratio calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.difference.info.formula": {
+        "value": "Primary - Secondary",
+        "comment": "The content describes the formula of the difference calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.info.example.title": {
+        "value": "Example",
+        "comment": "The title of the section that provides an example for the calculation",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.change.info.example": {
+        "value": "Sales change this year compared to last year = (This year sales - Last year sales) / Last year sales",
+        "comment": "The example illustrating the change calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.ratio.info.example": {
+        "value": "Sales this year compared to a quota = This year sales / quota",
+        "comment": "The example illustrating the ratio calculation.",
+        "limit": 0
+    },
+    "properties.comparison.calculationType.as.difference.info.example": {
+        "value": "...",
```

Comment:
Can you please point me to a place in UX specification that uses this text? Probably its `comment` need some more love because based on it I'm not sure how is this text used and how to translate it.

Reply:
There is a UI specification for this text:
https://www.figma.com/file/X5ENbAl731DRtUdR8hI02g?node-id=1228:24589&mode=design#527702227

As Petr Kaspar said in the [comment](https://www.figma.com/file/X5ENbAl731DRtUdR8hI02g?node-id=1228:24563&mode=design#521773806), this is only a temporary text.

Reply:
OK, in this case it is ok.